pkg/build/controller/build/build_controller.go

```diff
@@ -157,22 +157,24 @@ type BuildController struct {
 	buildConfigQueue      workqueue.RateLimitingInterface
 	controllerConfigQueue workqueue.RateLimitingInterface
 
-	buildStore                    buildv1lister.BuildLister
-	secretStore                   v1lister.SecretLister
-	podStore                      v1lister.PodLister
-	imageStreamStore              imagev1lister.ImageStreamLister
-	openShiftConfigConfigMapStore v1lister.ConfigMapLister
+	buildStore                      buildv1lister.BuildLister
+	secretStore                     v1lister.SecretLister
+	podStore                        v1lister.PodLister
+	imageStreamStore                imagev1lister.ImageStreamLister
+	openShiftConfigConfigMapStore   v1lister.ConfigMapLister
+	controllerManagerConfigMapStore v1lister.ConfigMapLister
```

Comment:
this is going to conflict with https://github.com/openshift/origin/pull/22734, can you rebase on top of it?

Reply:
will do.